src/main/java/seedu/address/logic/commands/ExitCommand.java

```diff
@@ -9,7 +9,7 @@ public class ExitCommand extends Command {
 
     public static final String COMMAND_WORD = "exit";
 
-    public static final String MESSAGE_EXIT_ACKNOWLEDGEMENT = "Exiting Address Book as requested ...";
+    public static final String MESSAGE_EXIT_ACKNOWLEDGEMENT = "Exiting CinternS as requested ...";
```

Comment:
I guess this ExitCommand is for the original Address Book application? Not important since it will eventually be deleted anyways

Reply:
Yeah I modified accidentally. Change it back anyway 😄